flixel/system/FlxSound.hx

```diff
@@ -383,7 +388,7 @@ class FlxSound extends FlxBasic
 	 * 
 	 * @param	ForceRestart	Whether to start the sound over or not.  Default value is false, meaning if the sound is already playing or was paused when you call play(), it will continue playing from its current position, NOT start again from the beginning.
 	 */
-	public function play(ForceRestart:Bool = false):FlxSound
+	public function play(ForceRestart:Bool = false, Position:Float = 0.0):FlxSound
```

Comment:
Adding `Position` here creates the awkward situation that the argument is not settable from the `SoundFrontEnd` function. We really should add any more arguments to those though if possible though, they already have too many.

It would be much better to implement a setter for `time`. That should be done separately though.